bundles/org.openhab.binding.dwdpollenflug/src/main/java/org/openhab/binding/dwdpollenflug/internal/handler/DWDPollenflugRegionHandler.java

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.dwdpollenflug.internal.handler;
+
+import static org.openhab.binding.dwdpollenflug.internal.DWDPollenflugBindingConstants.*;
+
+import java.util.Date;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.State;
+import org.openhab.binding.dwdpollenflug.internal.config.DWDPollenflugRegionConfiguration;
+import org.openhab.binding.dwdpollenflug.internal.dto.DWDPollenflug;
+import org.openhab.binding.dwdpollenflug.internal.dto.DWDRegion;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link DWDPollenflugRegionHandler} is the handler for bridge thing
+ *
+ * @author Johannes DerOetzi Ott - Initial contribution
+ */
+@NonNullByDefault
+public class DWDPollenflugRegionHandler extends BaseThingHandler implements DWDPollenflugRegionListener {
+
+    private final Logger logger = LoggerFactory.getLogger(DWDPollenflugRegionHandler.class);
+
+    private @NonNullByDefault({}) DWDPollenflugRegionConfiguration thingConfig = null;
+
+    private @Nullable DWDPollenflugBridgeHandler bridgeHandler;

```

Comment:
better do not cache the bridge handler but get it if you need it.

Reply:
Please check my fix!